charts/metering-ansible-operator/values.yaml

```diff
@@ -526,9 +526,10 @@ olm:
 
   subscriptionName: metering-ocp
   subscriptionChannel: stable
-  catalogSourceConfigName: installed-redhat-metering-operators-openshift-metering
-  catalogSourceConfigNamespace: openshift-marketplace
-  catalogSourceConfigTargetNamespace: openshift-metering
+  catalogSourceName: installed-redhat-metering-operators-openshift-metering
+  catalogSourceNamespace: openshift-marketplace
+  catalogSourceTargetNamespace: openshift-metering
+  catalogSourceSourceType: grpc
```

Comment:
We also need to emulate these `catalogSource*` keys in the upstream-values.yaml.

Reply:
done